automation/JJB/python_analysis.yaml

```diff
@@ -9,11 +9,11 @@
     parameters:
       - string:
           name: SONAR_KEY
-          default: sonarqube_pulp_python_full_analysis
+          default: sonarqube_python_full_analysis
```

Comment:
Slight nit: I recommend a consistant naming scheme, so this would be "sonarqube_python_analysis".